sktime/forecasting/hf_transformers_forecaster.py

```diff
@@ -36,9 +31,9 @@ class HFTransformersForecaster(BaseForecaster):
 
     Parameters
     ----------
-    model_path : str
-        Path to the huggingface model to use for forecasting. Currently,
-        Informer, Autoformer, and TimeSeriesTransformer are supported.
```

Comment:
we should give a list of supported str or arcihtectures.

Reply:
Got it! I will mention the architectures currently supported by sktime